R/ee_roll_stat.R

```diff
@@ -0,0 +1,61 @@
+
+
+
+
+#' ee_roll
```

Comment:
What is `ee_roll`

Reply:
its meant for rolling pixel level statistics on imageCollections (basically stacked rasters) from `GEE`. It actually isn't really necessary since we are doing everything locally and actually analyzing at the site level.

Nonetheless, It does need a better description if we are going to keep it around for later